Header header
string[] image_filepaths
geometry_msgs/Point endeffector_position
float64 gripper_width
float64[7] joint_angles